### ISSUE TITLE
New Consumer Groups (Kafka offset storage)

### ISSUE THREAD

**Issue by vitoravancini** [krallistic/kafka-prometheus-exporter#1]
From the readme I've assumed that new consumers we're also supported (consumers that store the offsets  at kafka itself and not at zookeeper). Am I missing something or only zookeeper-based consumers are beeing exposed at this moment?

**Comment by krallistic** [comment]
In the description it say: 
```
If you use old consumer (which store their offset in zookeeper), these two Metrics about consumergroups and their offset are also aviable:
```
But your are right, its a bit missleading. Sorry about that. Orginally my wishlist was to do also new offsets, but sadly didnt had the time&focus about that.

**Comment by vitoravancini** [comment]
Yeah I probably shouldve paid more attention to your readme. I'm currently using this exporter and another one for the new consumer offsets, it's not ideal but its working for now. Thank you @krallistic

**Comment by krallistic** [comment]
Out of curiosity, which one for new ones?

**Comment by vitoravancini** [comment]
this one: https://github.com/danielqsj/kafka_exporter
